.travis.yml

```diff
@@ -1,15 +1,6 @@
 language: generic
 conditions: v1
 
-branches:
-  only:
-  - stable
-  - dev
-  - /^release-.*$/
-  - /^prerelease-.*$/
-  - /^travis-.*$/
-  - /^import-.*$/
```

Comment:
i think there was a reason why we had this, which is to avoid travis to run always.

Reply:
@XVilka

Reply:
https://im.rizin.re/rizinorg/pl/6aunbdp58tyejmspmafepq491c